homeassistant/components/media_player/emby.py

```diff
@@ -178,6 +183,16 @@ def update(self):
         """Get the latest details."""
         self.update_devices(no_throttle=True)
         self.update_sessions(no_throttle=True)
+        # Check if we should update progress
+        try:
+            position = self.session['PlayState']['PositionTicks']
+        except (KeyError, TypeError):
+            position = None
```

Comment:
I would expect `media_status_last` and `media_status_received` to be set to `None` if the data is not there.

(I also think that the name `media_status_last` is ambiguous, what about `media_status_last_position` ?